### DISCUSSION TITLE
Rendering react-flow within react-flow

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#3381] [Q&A]
Hi,

I'm trying out an experience. I like the react-flow's zoom in and out, and I wanted to see what if I create a node that is an react-flow component.

Here is the sample on how it looks, but it is not rendering as expected. 
![Screenshot 2023-09-13 at 11 07 11 AM](https://github.com/wbkd/react-flow/assets/6542274/415aef9e-eb4b-4c19-92ea-83d03b82d9ff)

**Comment by bcakmakoglu** [comment]
Afaik this is not supported, due to some technical limitations 😞
